src/client/Microsoft.Identity.Client/MsalServiceExceptionFactory.cs

```diff
@@ -66,26 +67,37 @@ private static bool IsThrottled(OAuth2ResponseBase oAuth2Response)
                oAuth2Response.ErrorDescription.StartsWith(Constants.AadThrottledErrorCode);
         }
 
-        internal static MsalServiceException FromBrokerResponse(
```

Comment:
I refactored this method a bit. It does not need to receive so many params, just `msalResponse`

Reply:
Looks good